_run/common-commands.mk

```diff
@@ -55,6 +59,18 @@ provider-update:
 provider-status:
 	$(AKASH) provider status $(PROVIDER_ADDRESS)
 
+.PHONY: jwt-server-authenticate
+jwt-server-authenticate:
+	$(AKASH) provider jwt-server-authenticate \
+		--from      "$(KEY_ADDRESS)" \
+		--provider  "$(PROVIDER_ADDRESS)"
+
+.PHONY: run-jwt-server
+run-jwt-server:
+	$(AKASH) provider run-jwt-server \
```

Comment:
let's call this command `auth-server` - `akash provider auth-server`.

Reply:
Done